wrapanapi/systems/openstack.py

```diff
@@ -1059,3 +1069,133 @@ def usage_and_quota(self):
             'cpu_total': host_cpus,
             'cpu_limit': data['maxTotalCores'] if data['maxTotalCores'] >= 0 else None,
         }
+
+    def list_containers(self):
+        """List of available containers.
+
+        Returns: list List of containers name.
+        """
+
+        _, containers = self.sapi.get_account()
+        return [cont["name"] for cont in containers]
+
+    def container_exist(self, container_name):
```

Comment:
@digitronik, shouldn't we turn such stuff into entities as it was done for vms and etc ?
@mshriver, what's your opinion ?

Reply:
My view:
It is fastest way to check on `provider side`.
Yup, we must have this in entity means project side eg. (`cfme`).

Reply:
I would agree with @izapolsk  here, there should be a `SwiftContainer` (something like that) entity here, treated just like the templates and vms.  The object methods below should be methods on that entity class.

The System object here should just have methods like `list_containers` and `get_container`. Mirroring the pattern for methods concerning VM/Template entities.

Also, many of these methods are catching 404 and returning False, instead of raising an exception like I think they should. Look at the current patterns for `System.get_<thing>()` methods, which will raise `MutlipleInstancesError` or `VMInstanceNotFound` when the queried name doesn't exist, or exists as multiple items.

Reply:
@izapolsk @mshriver ok... I am removing check for exist of container and object.